src/runner/main.cpp

```diff
@@ -25,7 +24,6 @@
 #include <winrt/Windows.System.h>
 
 #if _DEBUG && _WIN64
-#include "unhandled_exception_handler.h"

```

Comment:
we need to revert it. this is still might be used, though currently commented, see `init_global_error_handlers`.